tools/lint-docstring

```diff
@@ -138,6 +177,32 @@ def extract_folder_file_structure() -> Dict[str, List[str]]:
     return folders_and_files
 
 
+def extract_docstrings_from_file_overview() -> Dict[str, Dict[str, str]]:
+    """
+    Reads the developer-file-overview.md document and creates
+    a dictionary containing folder name which in turn is a dictionary
+    containing files and their respective descriptons
+    """
+    with open(DEVELOPER_DOC, "r") as file:
+        doc_data = file.readlines()
+
+    computed_doc_dict: Dict[str, Dict[str, str]] = {}
+
+    # table is present from line 6 to (length - 1)
+    for i in range(TABLE_OFFSET - 1, len(doc_data)):
+        _, folder, filename, docstring, _ = str(doc_data[i]).split("|")
```

Comment:
Use the same slicing iterator you used in the other function here to keep the same style.
`for column in doc_data[TABLE_OFFSET-1:]`